examples/src/guides/expression_language/zep/zep_cloud_vector_store.ts

```diff
@@ -0,0 +1,77 @@
+import { ZepClient } from "@getzep/zep-cloud";
+import {
+  BasePromptTemplate,
+  ChatPromptTemplate,
+  PromptTemplate,
+} from "@langchain/core/prompts";
+import { ConsoleCallbackHandler } from "@langchain/core/tracers/console";
+import { ChatOpenAI } from "@langchain/openai";
+import { Document } from "@langchain/core/documents";
+import {
+  RunnableLambda,
+  RunnableMap,
+  RunnablePassthrough,
+} from "@langchain/core/runnables";
+import { ZepCloudVectorStore } from "@langchain/community/vectorstores/zep_cloud";
+import { formatDocument } from "langchain/schema/prompt_template";
+import { StringOutputParser } from "@langchain/core/output_parsers";
+
+const DEFAULT_DOCUMENT_PROMPT = PromptTemplate.fromTemplate("{pageContent}");
+
+async function combineDocuments(
+  docs: Document[],
+  documentPrompt: BasePromptTemplate = DEFAULT_DOCUMENT_PROMPT,
+  documentSeparator = "\n\n"
+) {
+  const docStrings: string[] = await Promise.all(
+    docs.map((doc) => formatDocument(doc, documentPrompt))
+  );
+  return docStrings.join(documentSeparator);
+}
+
+// Your Zep Collection Name
+const collectionName = "<Zep Collection Name>";
+
+const zepClient = new ZepClient({
+  // Your Zep Cloud Project API key https://help.getzep.com/projects
+  apiKey: "<Zep Api Key>",
+});
+
+const vectorStore = await ZepCloudVectorStore.init({
+  client: zepClient,
+  collectionName,
+});
+
+const prompt = ChatPromptTemplate.fromMessages([
+  [
+    "system",
+    `Answer the question based only on the following context: {context}`,
+  ],
+  ["human", "{question}"],
+]);
+
+const model = new ChatOpenAI({
+  temperature: 0.8,
+  modelName: "gpt-3.5-turbo-1106",
+});
+const retriever = vectorStore.asRetriever();
+
+const setupAndRetrieval = RunnableMap.from({
+  context: new RunnableLambda({
```

Comment:
No big deal, but you don't need to wrap in the `RunnableLambda` constructor. Raw functions will be coerced.